tests/test_server_pool.py

```diff
@@ -374,6 +375,7 @@ def on_disconnect(self):
 
 class SimulatedCaseMeta(type):
     def __new__(mcls, name, bases, dct):
+        debug_pool = os.environ.get('EDGEDB_TEST_DEBUG_POOL')
```

Comment:
Why not just make this check in `setUp`?

Reply:
That makes a lot more sense! Fixed now.